BetaCameras/Hikvision/Hikvision.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Copyright (c) Metrilus GmbH
+// MetriCam 2 is licensed under the MIT license. See License.txt for full license text.
+
+using MetriCam2.Exceptions;
+using Metrilus.Util;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Net;
+using System.Threading;
+#if NETSTANDARD2_0
+#else
+using System.Drawing.Imaging;
+#endif
+
+
+namespace MetriCam2.Cameras
+{
+    public class Hikvision : Camera, IDisposable
+    {
+        private bool _disposed = false;
+        private RTSPClient _client = null;
+        private long _currentBitmapTimestamp = 0;
+        private Bitmap _currentBitmap = null;
+
+        ParamDesc<string> IPAddressDesc
+        {
+            get
+            {
+                ParamDesc<string> res = new ParamDesc<string>
+                {
+                    Description = "IP address of the camera",
+                    ReadableWhen = ParamDesc.ConnectionStates.Connected | ParamDesc.ConnectionStates.Disconnected,
+                    WritableWhen = ParamDesc.ConnectionStates.Disconnected
+                };
+                return res;
+            }
+        }
+        public string IPAddress { get; set; }
+
+        ParamDesc<uint> PortDesc
+        {
+            get
+            {
+                ParamDesc<uint> res = new ParamDesc<uint>
+                {
+                    Description = "Port of RTSP",
+                    ReadableWhen = ParamDesc.ConnectionStates.Connected | ParamDesc.ConnectionStates.Disconnected,
+                    WritableWhen = ParamDesc.ConnectionStates.Disconnected
+                };
+                return res;
+            }
+        }
+        public uint Port { get; set; }
+
+        ParamDesc<string> UsernameDesc
+        {
+            get
+            {
+                ParamDesc<string> res = new ParamDesc<string>
+                {
+                    Description = "Username to access the camera",
+                    ReadableWhen = ParamDesc.ConnectionStates.Connected | ParamDesc.ConnectionStates.Disconnected,
+                    WritableWhen = ParamDesc.ConnectionStates.Disconnected
+                };
+                return res;
+            }
+        }
+        public string Username { get; set; }
+
+        ParamDesc<string> PasswordDesc
+        {
+            get
+            {
+                ParamDesc<string> res = new ParamDesc<string>
+                {
+                    Description = "Password to access the camera",
+                    ReadableWhen = ParamDesc.ConnectionStates.Connected | ParamDesc.ConnectionStates.Disconnected,
+                    WritableWhen = ParamDesc.ConnectionStates.Disconnected
+                };
+                return res;
+            }
+        }
+        public string Password { get; set; }
```

Comment:
I would set an empty string to avoid problems with the designer.
```suggestion
        public string Password { get; set; } = "";
```